src/main/java/org/cascadebot/cascadebot/commands/management/permission/UserPermissionRemoveSubCommand.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.management.permission;
+
+import java.util.Set;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.commandmeta.Argument;
+import org.cascadebot.cascadebot.commandmeta.ArgumentType;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.permissions.PermissionsManager;
+import org.cascadebot.cascadebot.permissions.objects.User;
+import org.cascadebot.cascadebot.utils.DiscordUtils;
+
+public class UserPermissionRemoveSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if(context.getArgs().length < 2) {
+            context.getUIMessaging().replyUsage(this, "userperms");
+            return;
+        }
+
+        Member member = DiscordUtils.getMember(context.getGuild(), context.getArg(0)); //TODO Switch these of over to the error handler in language
```

Comment:
> TODO Switch these of over to the error handler in language
I'm assuming here you're just talking about the responses section of lang?

Reply:
Yep